tidb-binlog/tidb-binlog-relay-log.md

```diff
@@ -52,6 +52,8 @@ While Drainer is running, if it confirms that the whole data of a relay log file
 
 If the size of a relay log file reaches 10MB (by default), the file is split, and data is written into a new relay log file.
```

Comment:
I think you need to update L50-L53, rather than add a new paragraph.

Reply:
Removed in https://github.com/pingcap/docs/pull/4218/commits/9271fc6b85c555e69c58d5f91b01d4ba1dfe8cbe. I forgot to delete these lines.